src/pages/lessons/[slug].tsx

```diff
@@ -303,7 +303,7 @@ const Lesson: FunctionComponent<LessonProps> = ({initialLesson}) => {
                         player={playerRef}
                         url={transcript_url}
                         fetcher={fetcher}
-                        send={send}
+                        playVideo={() => send('PLAY')}
```

Comment:
since we don't need to control the entire state machine, send in a specific method to do just what we want